django_project/changes/feeds/sponsor.py

```diff
@@ -163,6 +170,18 @@ class AtomSponsorFeed(RssSponsorFeed):
     feed_type = Atom1Feed
     subtitle = RssSponsorFeed.description
 
+class AtomPastSponsorFeed(RssPastSponsorFeed):
+    """Atom Feed class for past sponsor."""
+
+    feed_type = Atom1Feed
+    subtitle = RssPastSponsorFeed.description
+
+class AtomPastSponsorFeed(RssPastSponsorFeed):
```

Comment:
__[Flake8]__

[F811] redefinition of unused 'AtomPastSponsorFeed' from line 173


<!-- Comment by [SideCI](https://sideci.com) -->